layouts/partials/mailing-list.html

```diff
@@ -42,10 +42,8 @@ <h2 class="major">{{ i18n "subscribe-title" }}</h2>
                 <div class="submit-button-container col-xs-12 col-sm-4">
                   <button type="submit" 
                         id="mc-embedded-subscribe"
-                        class="button">{{ i18n "subscribe" }}<object type="image/svg+xml" tabindex="-1" role="img"
```

Comment:
You may also be able to put the `pointer-events: none;` style on the `#mc-embedded-subscribe object` selector and leave as an svg.